deploy/examples/create-external-cluster-resources.py

```diff
@@ -826,7 +829,44 @@ def validate_rados_namespace(self):
             raise ExecutionFailureException(
                 ("The provided rados Namespace, '{}', is not found in the pool '{}'").format(
                     rados_namespace, rbd_pool_name))
-
+    
+    def get_rgw_fsid(self):
+        access_key = self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY']
+        access_key = "WC2D69F9L7KEOJVETKER"
```

Comment:
why access key and secret key have hardcoded values?

Reply:
Yaa removed it, added mistakenly test cluster info